app/scripts/templates/settings/security.mustache

```diff
@@ -0,0 +1,87 @@
+        {{/hasToken}}
+    </div>
+
+
+    <div class="settings-unit-details">
+        <form novalidate>
+            <p>
+                {{#t}}Add an extra layer of security to your account by requiring security codes.{{/t}}
+            </p>
+
+            <ul id="security-status" class="security-list">
+                <li class="security-type">
+                    <div class="name">Current status</div>
+                    <div class="details">
+                        {{#hasToken}}
+                            <div class="enabled">{{#t}}Enabled{{/t}}</div>
+                        {{/hasToken}}
+                        {{^hasToken}}
+                            <div class="disabled">{{#t}}Disabled{{/t}}</div>
+                        {{/hasToken}}
+                    </div>
+                    <div class="settings-button-group">
+                        {{#hasToken}}
+                            <button class="settings-button warning totp-delete">{{#t}}Disable{{/t}}</button>
+                        {{/hasToken}}
+                        {{^hasToken}}
+                            <button class="settings-button secondary totp-create">{{#t}}Enable{{/t}}</button>
+                        {{/hasToken}}
+                    </div>
+                </li>
+                {{#hasToken}}
+                    <!-- This will be enabled in https://github.com/mozilla/fxa-content-server/issues/5865 -->
+                    <!--<a class="show-recovery-codes">{{#t}}Show recovery codes{{/t}}</a>-->
+                {{/hasToken}}
+            </ul>
+
+            <div id="totp" class="totp-details hidden">
+                <div class="qr-image-container">
+                    <img class="qr-image"/>
+                </div>
+
+                <p class="setup-description">
+                    {{#t}}Scan the QR code in your app and confirm it is setup correctly by entering the security code it provides.{{/t}}
+                    <a class="show-code-link">{{#t}}Can't scan code?{{/t}}</a>
+                </p>
+
+                <div class="manual-code hidden">
+                    <p class="setup-description">{{#t}}Manually enter this secret key into your authentication app:{{/t}}</p>
+                    <div class="code"></div>
+                </div>
+
+                <div class="input-row">
+                    <label class="label-helper"></label>
+                    <input type="text" class="text totp-code tooltip-below" placeholder="{{#t}}Security code{{/t}}" value="{{ code }}" required autofocus  autocomplete="off"/>
+                </div>
+            </div>
+
+            <div class="button-row">
+                {{#hasToken}}
+                    <button type="submit" class="settings-button primary totp-refresh">{{#t}}Refresh{{/t}}</button>
+                {{/hasToken}}
+
+                {{^hasToken}}
+                    <button type="submit" class="settings-button primary totp-confirm-code">{{#t}}Confirm{{/t}}</button>
```

Comment:
If I refresh the page when it is with a QR code the confirm button still shows in blue but there is nothing to confirm:

<img src=https://user-images.githubusercontent.com/128755/37054948-6b4d1e16-214e-11e8-82fe-13f8d3bcba1a.png width=390 />